user-api/src/main/java/com/sleepwell/userapi/reservation/entity/Reservation.java

```diff
@@ -0,0 +1,66 @@
+package com.sleepwell.userapi.reservation.entity;
+
+import com.sleepwell.userapi.accommodation.entity.Accommodation;
+import com.sleepwell.userapi.member.entity.Member;
+import com.sleepwell.userapi.reservation.dto.ReservationDetailInfoDto;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDate;
+
+@Getter
+@Setter
+public class Reservation {
+
+    private Long id;
+
+    private String paymentType;
+
+    private LocalDate checkInDate;
+
+    private LocalDate checkOutDate;
+
+    private int numberOfGuest;
+
+    private String reservationStatus;
+
+    private Member guest;
+
+    private Accommodation accommodation;
+
+    @Builder
+    public Reservation(String paymentType, LocalDate checkInDate, LocalDate checkOutDate, int numberOfGuest) {
+        this.paymentType = paymentType;
+        this.checkInDate = checkInDate;
+        this.checkOutDate = checkOutDate;
+        this.numberOfGuest = numberOfGuest;
+    }
+
+    public void createReservation(Member guest, Accommodation accommodation) {
```

Comment:
이 메소드는 이름과 반환값이 잘 안맞는 것 같아요.
```suggestion
    public Reservation createReservation(Member guest, Accommodation accommodation) {
```
이거나
```suggestion
    public void updateReservation(Member guest, Accommodation accommodation) {
```
이어야 하지 않을까요?

사실 그것보다 더 신경써야하는 부분은 Reservation클래스에서 createReservation이 있는데, 이 메소드가 static도 아니고, Reservation을 반환하지도 않는 create메소드라는 것인데 .... 이 부분에 대해서도 생각해 보시면 좋을 것 같습니다.

Reply:
`updaetReservation`으로 변경해야 할 것 같습니다.
메소드 내용 상 Reservation과 연관관계를 맺고있는 것들을 함께 갱신해주는 작업을 수행하는데, create라는 이름과는 맞지 않는 것 같습니다. 수정하겠습니다.